docs/implement-check.md

```diff
@@ -0,0 +1,70 @@
+# Integrating Custom Check Into The Tool
+
+The Static Analysis Tool provides the necessary infrastructure for custom check implementations using the API of FindBugs, Checkstyle and PMD. 
+
+Helpful links when writing a custom check for the first time may be:
+
+- for [Checkstyle](http://checkstyle.sourceforge.net/writingchecks.html);

```

Comment:
Why don't you use directly a link to the anchor for writing checks http://checkstyle.sourceforge.net/writingchecks.html#Writing_Checks ?